### PR TITLE
utils: add fallback mimetype

### PR DESCRIPTION
**Pull request by jrcastro2**
* closes https://github.com/zenodo/rdm-project/issues/547

### CODE REVIEW

File: invenio_files_rest/utils.py [hunk 1]
```diff
@@ -21,6 +21,9 @@
 }
 """Mapping encoding to MIME types which are not in mimetypes.types_map."""
 
+FALLBACK_MIMETYPE = "application/octet-stream"
```
Comment (kpsherva) [inline invenio_files_rest/utils.py:24]: does it make sense to expose it as config ? ping @slint

Comment (slint) [reply invenio_files_rest/utils.py]: Actually, this doesn't make sense to have as config, the fallback for "I've no idea what I'm serving" is `application/octet-stream` (i.e. "you get bytes").

**Comment by jrcastro2** [comment]
Not required anymore as we agreed to hardcode the default mimetype

**Comment by slint** [comment]
Just to clarify, I'm against moving this to config since IMHO it's a constant, and I also agree that introducing it in `utils.py` as `FALLBACK_MIMETYPE` might have been a "brittle" solution.

**Pull request closed without merging**